### PR TITLE
Add more changes to tool.

### LINKED ISSUE #32: Also detect constants changed in dart:_http.
The CL https://dart-review.googlesource.com/c/sdk/+/58620 changed more constants to camelCase. These should also be fixed by the script.

### PR DESCRIPTION
**Pull request by lrhn**
Fixes #32.

### CODE REVIEW

**Comment by kevmoo** [comment]
@lrhn – bump the changelog and version in the pubspec?

File: lib/src/changes.dart [hunk 1]
```diff
@@ -207,6 +207,145 @@ final Map<String, String> _textReplacements = {
   // dart:io/string_transformer.dart
   'SYSTEM_ENCODING': 'systemEncoding',
 
+  // dart:_http/http.dart
+  'HttpStatus.CONTINUE': 'HttpStatus.continue_',
```
Comment (devoncarew) [inline lib/src/changes.dart:181]: Does this constant really end with an underscore?

**Comment by devoncarew** [comment]
If this PR fixes issue 32, in the PR description above, you can write `fixes #32`, `fix #32` or `closes #32`, and when the PR is merged, the associated issues will automatically be closed.

**Review by devoncarew — COMMENTED** [review]
Can you also add some smoke tests for the associated constant changes (see the existing tests)?

File: lib/src/changes.dart (hunk 1 above)
Comment (lrhn) [reply lib/src/changes.dart:181]: Seems so. Reserved words are annoying.

**Comment by lrhn** [comment]
PTAL

**Review by devoncarew — APPROVED** [review]

**Comment by devoncarew** [comment]
@lrhn, I added you as an uploader for this package - feel free to re-publish once this PR lands.

**Pull request merged**